my-dsl.md

```diff
@@ -2,37 +2,66 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[POV-Ray](http://www.povray.org/)
+([Source-code](https://github.com/POV-Ray/povray))
 
 # Domain
 _Describe the language's domain in five words._
 
+Creates images using ray tracing
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+POV-Ray contains a _front-end_ and a _back-end_.  A user specifies various
+fields such as a light source, camera, shape, background, etc.  The front-end
+locates the files and gives them to the back-end to parse and compute the
+image data.  The back-end then sends the information back to the front-end,
+which writes the image data to a file. [Source](https://github.com/POV-Ray/povray/blob/master/source-doc/architecture.md)
 
 # DSL-ness
-_Fowler writes about a spectrum of languages, from general-purpose languages to 
-"purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
+_Fowler writes about a spectrum of languages, from general-purpose languages to
+"purely" domain-specific. Where does the DSL you chose fall on this spectrum,
 and why?_ 
 
+POV-Ray is closer to the domain-specific side, since it is purely used to 
+generate images.  However, I would hesitate to say that it is "purely"
+domain-specific, because it supports loops and macros that make it
+Turing-complete.  Regardless, the intended use is for creating graphics.
```

Comment:
Turing Completeness is a bad metric for whether something is a DSL or a GPL. As it supports more language features it is certainly further to the GPL side, but in general as long as something is designed for a domain it's a DSL (obviously there's more to it than that, but that should be your first clue).